internal/osbuild2/kickstart_stage.go

```diff
@@ -44,7 +44,8 @@ func NewKickstartStageOptions(
 	userCustomizations []blueprint.UserCustomization,
 	groupCustomizations []blueprint.GroupCustomization,
 	ostreeURL string,
-	ostreeRef string) (*KickstartStageOptions, error) {
+	ostreeRef string,
+	osName string) (*KickstartStageOptions, error) {
```

Comment:
Noticed this change in the callers before seeing it here and figuring out why it happened.  If there's a rebase at any point for whatever reason, could you please mention it in the commit message?

Reply:
Acknowledged. I have updated the description to explain the change in the function signature.